### ISSUE TITLE
[BUG] Ecran rotatif - non fonctionnel

### ISSUE THREAD

**Issue by pppplus** [pascalaubry/papi-web#59]
## Description du bug
Les écrans rotatifs ne fonctionnent pas
On voit bien le petit logo s'afficher pour la rotation de page, mais la fenêtre reste sur le premier tournoi
J'avais bien plusieurs tournois sur la fenêtre

## Environnement d'utilisation
- Windows 11
- Version de Papi-web : 2.4.16

**Comment by Amaras** [comment]
Bonjour, merci de la remontée de problème.
Pourriez-vous envoyer le fichier `.db` (sans les mots de passe) que vous utilisez pour qu'on essaie de reproduire le bug ?
Il se trouve dans le dossier `events/` de l'installation papi-web, et a pour nom l'identifiant de l'événement que vous avez paramétré.

**Comment by pppplus** [comment]
Tu parles des mots de passe de chaque tournoi, pour envoyer sur le site FFE ?

**Comment by Amaras** [comment]
Exactement, et éventuellement des mots de passe ChessEvent (si utilisation de cette fonctionnalité)

**Comment by Amaras** [comment]
Bonjour @pppplus, est-ce que vous avez le fichier de votre événement à nous mettre à disposition ?
Ou alors aviez-vous trouvé comment corriger une éventuelle erreur de configuration de votre côté ?

Si je n'ai pas de nouvelles avant le nouvel an, je vais fermer ce rapport de bug pour non-reproduction du comportement.

**Comment by Amaras** [comment]
Je ferme donc cette issue, n'hésitez pas à la rouvrir pour nous envoyer le fichier.
